### ISSUE TITLE
GitPod: add installation of Cypress tests to workspace startup file

### ISSUE THREAD

**Issue by ieahleen** [freeCodeCamp/freeCodeCamp#43838]
The instructions in this part of the guide https://contribute.freecodecamp.org/#/how-to-add-cypress-tests?id=cypress-gitpod-setup
Could be added to the GitPod startup file instead of having to do it manually.

@ShaunSHamilton is it possible to do it now? Or the feature is still in preview?

**Comment by ShaunSHamilton** [comment]
@ieahleen Yes, it is possible. However, it adds about 5-7 mins to the start-up of the environment (when you click the GitPod button on a PR).

So, we specifically did not add this.

Also, I am concerned it would not work as expected, because the setup requires a keyboard to be selected. I do not fully understand why, but if this is done automatically, certain tests might not behave as expected.
